docs/firebase_layers.js

```diff
@@ -62,6 +62,7 @@ function createContinuousFunction(field, minVal, maxVal, color) {
   const range = maxVal - minVal;
   return (feature) => {
     let value = feature['properties'][field];
+    if (!value) return transparent;
```

Comment:
should probably check explicitly for null. What if we have a range in which 0 is valid? Same for below

Reply:
Ah good point